spec/components/officing/results/form_component_spec.rb

```diff
@@ -16,4 +16,13 @@
     expect(page).to have_field "Invalid ballots", with: 0, type: :number
     expect(page).to have_field "Valid ballots", with: 0, type: :number
   end
+
+  it "do not renders open-ended questions" do
+    create(:poll_question_open, poll: poll, title: "What do you want?")
+
+    render_inline Officing::Results::FormComponent.new(poll, Poll::OfficerAssignment.none)
+
+    expect(page).not_to have_css "fieldset", text: "What do you want?"
```

Comment:
To make sure this text isn't anywhere, not just instide a <fieldset> tag:

```suggestion
    expect(page).not_to have_content "What do you want?"
```